### ISSUE TITLE
Enhancement: Can you add a global Title lock to images also. (see #64)

### ISSUE THREAD

**Issue by prcoy** [MatMoul/quickdial-webext#127]
Here's the situation. It seems every time Firefox updates, (about every two weeks it seems!) I loose both titles _and_ images on all my Quickdial bookmarks. Given I have about 250-300 of them, it take a couple of days to get most of them back to "useful". The worst part is all the youtube bookmarks revert to a generic title of "YouTube" with a blank white image. ugh. I have to do a "capture in a new tab" on each of them.
Anyway, I noticed that titles and images are individually lockable BUT a global lock on images and titles would be a godsend.

**Comment by MatMoul** [comment]
It's very strange that you encounter the issue each 2 weeks or FF updates...
It seem you have a conflict with an other extention...
But for the global title lock, I'll try to provide a solution

**Comment by BillZM** [comment]
A good first step would be to default to locking the title when one edits it. Perhaps this could be a behavior one toggles on or off, but it should default to on.
In my case, I have edited titles, then saved bookmarks and imported to other computers only to have to re-edit the same titles. I'm assuming locked titles would copy over with Firefox backup and restore of bookmarks.

**Comment by MatMoul** [comment]
@BillZM Yeah but I'm missing time to work on this project, my job is very hard this year...
anyone is welcome to contribute or fork the project...
